charts/pega/charts/constellation/templates/clln-deployment.yaml

```diff
@@ -0,0 +1,36 @@
+#Deploy only when the constellation flag has been enabled in the values yaml.
+{{ if and .Values.enabled (eq .Values.enabled true) }}
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: constellation
+  labels:
+    app: constellation
+    # component: constellation
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: constellation
+      # component: constellation
+  template:
+    metadata:
+      labels:
+        app: constellation
+        # component: constellation
+    spec:
+      imagePullSecrets:
+      - name: regcred
+      containers:
+      # cli - the sidecar - here be the brains
+      - name: clln-cli
+        #imagePullPolicy: Never
```

Comment:
I believe there is an imagepullpolicy that is available from the parent charts.

Reply:
Yes @dcasavant we will remove that commented policy and we will also take out line 25.

Reply:
TODO : add and define image pull policy for constellation chart.

Refer _pega-deployment.tpl